python/packages/ai/teams/ai/planner/planner.py

```diff
@@ -58,3 +63,41 @@ def add_function(
             allow_overrides (bool, optional): Whether to allow overriding an existing function.
             Defaults to False.
         """
+
+    @abstractmethod
+    async def review_prompt(
```

Comment:
The moderator functionality may be decoupled from planner like other SDK? As developers can choose a different moderator and even an empty moderator for their app.

Reply:
No we are consolidating the moderator and planner since for each planner there is a cooresponding moderator

Reply:
this change will also be made in the other SDK's later on.